i18n/messages.se.js

```diff
@@ -41,5 +41,11 @@ module.exports = {
     template_silent_login_page_heading: 'Sida med tyst inloggning',
     template_not_found_page_heading: 'Sidan hittades inte ',
     template_back_link: 'Tillbaka till start-sidan',
+
+    language_link_lang_en: 'English',
+    language_link_not_translated: 'This page isn’t translated',
+    language_link_button_close: 'Stäng',
```

Comment:
Should the button label be swedish?
We are on the swedish page, but the rest of this dialog is in english.